examples/FFPB_KPFM/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Download Hartree Potentials"
+wget --no-check-certificate "https://zenodo.org/records/10563098/files/KPFM_hartree.tar.gz"
+
+echo "Extract Hartree Potentials"
+tar xzvf KPFM_hartree.tar.gz
```

Comment:
mistake here: `tar -xzvf KPFM_hartree.tar.gz`

Reply:
Thanks, I have changed that. It worked for me without the dash too, but it was probably a non-standard syntax.